### PR TITLE
Added Security category API

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/resources/api.ts [hunk 1]
```diff
@@ -39,6 +43,7 @@ export interface API {
   datapoints: DatapointsAPI;
   events: EventAPI;
   files: FileAPI;
+  securitycategories: SecurityCategoriesAPI;
```
Comment (polomani) [inline src/resources/api.ts]: I would suggest `securityCategories`

File: src/resources/api.ts [hunk 2]
```diff
@@ -60,6 +65,11 @@ export function generateAPIObject(
     datapoints: generateDatapointsObject(project, axiosInstance, metadataMap),
     events: generateEventsObject(project, axiosInstance, metadataMap),
     files: generateFilesObject(project, axiosInstance, metadataMap),
+    securitycategories: generateSecuritycategoryObject(
```
Comment (polomani) [inline src/resources/api.ts]: `generateSecurityCategoryObject`

File: src/resources/apikeys.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Cognite AS
```
Comment (polomani) [inline src/resources/apikeys.ts]: shouldn't this be an `assets.ts`?

File: src/resources/securitycategories.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Cognite AS
```
Comment (polomani) [inline src/resources/securitycategories.ts]: filename camel case?

File: src/resources/securitycategories.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+   * [Delete security categories](https://doc.cognitedata.com/api/v1/#operation/deleteSecurityCategories)
+   *
+   * ```js
+   * await client.securitycategories.delete([123, 456]);
+   */
+  delete: (ids: CogniteInternalId[]) => Promise<{}>;
+}
+
+/** @hidden */
+export function generateSecuritycategoryObject(
```
Comment (polomani) [inline src/resources/securitycategories.ts]: capital C

**Review by polomani — CHANGES_REQUESTED** [review]

**Comment by polomani** [comment]
please, rebase also

File: src/resources/apikeys.ts (hunk 3 above)
Comment (ghost) [reply src/resources/apikeys.ts]: Ops. This should be deleted.

**Review by polomani — APPROVED** [review]

**Pull request merged**